commands/api.go

```diff
@@ -222,7 +223,7 @@ func apiCommand(cmd *Command, args *Args) {
 			fi, err := os.Open(fn)
 			utils.Check(err)
 			body = fi
-			defer fi.Close()
+			defer utils.Check(fi.Close())
```

Comment:
If we couldn't close a file for whatever reason, I think that it would be a much better idea to allow the program to continue (it's not a big deal, after all) rather than to halt it.